backend/monitoring/dashboard.py

```
@@ -1404,11 +1405,27 @@ async def get_available_test_suites():
                     data={"test_suites": [], "total_suites": 0}
                 )
             except Exception as query_error:
-                # Any other query error (like column not found)
-                logger.error(f"Query error: {query_error}")
+                # Any other query error (like column not found) - provide detailed error for debugging
+                error_details = str(query_error)
```

Ruff (0.12.2)

Local variable `error_details` is assigned to but never used

Remove assignment to unused variable `error_details`

(F841)